specifyweb/frontend/js_src/lib/localization/interactions.ts

```diff
@@ -47,6 +47,9 @@ export const interactionsText = createDictionary({
     'ru-ru': 'Не обнаружено препаратов для следующих номеров по каталогу:',
     'uk-ua': 'Не знайдено препаратів для наступних каталожних номерів:',
   },
+  preparationsNotAvailableFor: {
+    'en-us': 'No preparations are available for the following Catalog Numbers:',
```

Comment:
I see that this string is based on the `preparationsNotFoundFor`, so they both have the same issue
- try hard not to use database field names/table names in UI localization as database field names/table are edited by collection managers, so this would create inconsistency/confusion.

Refactor to either pass field name as an argument (see how it's done in other places in the code - you did the refactor for this), or reword the sentence to not have to include the field name (i.e use `...for the following records:`